data/distro.yml

```
@@ -542,3 +542,25 @@
       text: '
         <p>To complete setup, restart your system. Now all you have to do is <a href="https://flathub.org/">install some apps</a>!</p>
         <p>Note: graphical installation of Flatpak apps may not be possible with GNU Guix.</p>'
+      </li>
+    </ol>
+
+- name: Crystal Linux
+  logo: "crystallinux.svg"
+  introduction: >
+    <h2>Flatpak is installed by default on Crystal Linux.<h2>
+    <p>If you didn't use jade_gui to install crystal or selected not to install it, you can set Flatpak up by using the following steps.<p>
+  steps:
+    - name: Install Flatpak
+      text: "
+        <p>To install Flatpak in Crystal Linux, you must first make sure your packages are up to date, run the following in a terminal:</p>
```

```suggestion
        <p>To install Flatpak in Crystal Linux, you must first make sure your packages are up to date. Run the following in a terminal:</p>
```

One more small thing.

---

@axtloss